### ISSUE TITLE
Make Background on Tooltips Darker

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/Pilcrow#1909]
The foreground and background colors of tooltips have a color contrast ratio that just barely passes WCAG AA at 4.6:1. This contrast should be increased, preferably so that it passes WCAG AAA or a ratio of 7:1. 

**Review Checklist**

* The color contrast of tooltips has been increased
* Documentation has been updated to reflect new changes

**Accessibility Checklist**

* No new color contrast issues are introduced

**Comment by sattler9** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pilcrow-Development-5ff39ba363391e0019713c87/issues/MESH-Research/Pilcrow/1909?planning-poker) with Zenhub @gmeben @vilagshe @wreality
